src/playground/service.clj

```diff
@@ -48,6 +52,25 @@
   {:status 200
    :body   {:temperature temperature :orientation orientation}})
 
+;;auth interceptors
+
```

Comment:
matched this to the db but not sure how to relate to `username` and `password` as they are values in `form-params` I think that the `(param-spec-interceptor :form-params)` brings forward the form-params . I will look again.